src/components/run-button/run-button-container.js

```diff
@@ -22,51 +22,53 @@ import {
     startVoltageInit,
     stopVoltageInit,
     stopLoadFlow,
-} from '../utils/rest-api';
-import { RunningStatus } from './utils/running-status';
+} from '../../utils/rest-api';
+import { RunButtonType, RunningStatus } from '../utils/running-status';
 
-import ContingencyListSelector from './dialogs/contingency-list-selector';
+import ContingencyListSelector from '../dialogs/contingency-list-selector';
 import {
     addLoadflowNotif,
     addSANotif,
     addSensiNotif,
     addShortCircuitNotif,
     addDynamicSimulationNotif,
     addVoltageInitNotif,
-} from '../redux/actions';
+    setRunButtonStatus,
+} from '../../redux/actions';
 import { useDispatch, useSelector } from 'react-redux';
 import { useIntl } from 'react-intl';
 import { useSnackMessage } from '@gridsuite/commons-ui';
-import { PARAM_DEVELOPER_MODE } from '../utils/config-params';
-import { useParameterState } from './dialogs/parameters/parameters';
+import { PARAM_DEVELOPER_MODE } from '../../utils/config-params';
+import { useParameterState } from '../dialogs/parameters/parameters';
 import DynamicSimulationParametersSelector, {
     checkDynamicSimulationParameters,
-} from './dialogs/dynamicsimulation/dynamic-simulation-parameters-selector';
+} from '../dialogs/dynamicsimulation/dynamic-simulation-parameters-selector';
 
 export function RunButtonContainer({
     studyUuid,
     currentNode,
-    loadFlowStatus,
-    securityAnalysisStatus,
-    sensiStatus,
-    shortCircuitStatus,
-    dynamicSimulationStatus,
-    voltageInitStatus,
     setIsComputationRunning,
-    runnable,
     disabled,
 }) {
-    const [loadFlowStatusState, setLoadFlowStatusState] =
-        useState(loadFlowStatus);
-    const [securityAnalysisStatusState, setSecurityAnalysisStatusState] =
-        useState(securityAnalysisStatus);
-    const [sensiStatusState, setSensiStatusState] = useState(sensiStatus);
-    const [shortCircuitStatusState, setShortCircuitStatusState] =
-        useState(shortCircuitStatus);
-    const [dynamicSimulationStatusState, setDynamicSimulationStatusState] =
-        useState(dynamicSimulationStatus);
-    const [voltageInitStatusState, setVoltageInitStatusState] =
-        useState(voltageInitStatus);
+    const loadFlowStatusState = useSelector(
+        (state) => state.runButtonStatus[RunButtonType.LOADFLOW]
+    );
+
+    const securityAnalysisStatusState = useSelector(
```

Comment:
I would remove the `State` at the end of the status

Reply:
done